packages/web-components/src/components/ic-accordion/ic-accordion.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Component,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+} from "@stencil/core";
+import { isSlotUsed } from "../../utils/helpers";
+import chevronIcon from "../../assets/chevron-icon.svg";
+import { IcThemeForeground, IcSizes } from "../../utils/types";
+
+let accordionIds = 0;
+@Component({
+  tag: "ic-accordion",
+  styleUrl: "ic-accordion.css",
+  shadow: true,
+})
+export class Accordion {
+  private accordionId = `ic-accordion-${accordionIds++}`;
+
+  @Element() el: HTMLIcAccordionElement;
+  /**
+   * The section header outlining section content.
+   */
+  @Prop() heading?: string = "";
+  /**
+   * The main body message of the card.
+   */
+  @Prop() message?: string = "";
+  /**
+   * If true accordion appears expanded.
+   */
+  @Prop({ mutable: true }) expanded: boolean = false;
+  /**
+   * @internal Determines whether the light or dark variant of the tabs should be displayed.
+   */
+  @Prop() appearance?: IcThemeForeground = "default";
+  /**
+   * The size of the accordion.
+   */
+  @Prop() size?: IcSizes = "default";
+  /**
+   * If `true`, the card will be disabled if it is clickable.
+   */
+  @Prop() disabled?: boolean = false;
+
+  @Event() accordionClicked: EventEmitter<{ id: string }>;
```

Comment:
Is this an event we want people to be able to access? If so, it will need to be renamed to icAccordionClicked. If not, it will need an @internal.